readme.md

```diff
@@ -200,104 +107,224 @@ $ adb forward tcp:9222 localabstract:chrome_devtools_remote
 $ lighthouse --disable-device-emulation --disable-cpu-throttling https://mysite.com
 ```
 
-## Tests
+## Using programmatically
 
-Some basic unit tests forked are in `/test` and run via mocha. ESLint also checks for style violations.
+The example below shows how to setup and run Lighthouse programmatically as a Node module.
```

Comment:
How important is supporting use as a node module to us? If we think it's important, any issues to make basic usage easier than 80 lines? If not, maybe let's move this to separate file and link to it from README.

Personally, I'd love to see chrome launcher published to NPM separately with a slightly friendlier interface (happy to take this up if there's agreement), and a more organized programmatic API that doesn't involve so much digging into our directory tree (more resilient to doing our rename of `lighthouse-cli -> cli` etc